docs/source/guide/zne-1-intro.myst

```diff
@@ -0,0 +1,125 @@
+
+Both steps — noise scaling and extrapolation — can be applied with Mitiq. The corresponding sub-modules are `mitiq.zne.scaling` and `mitiq.zne.inference`.
+
+The following code is compatible with the multiple frontends. The next code cell can be used for selecting one of them.
+
+```{code-cell} ipython3
+frontend = "cirq"  # Supported: "cirq", "qiskit", "pyquil", "braket", "pennylane".
+```
+
+## Define the circuit of interest
+
++++
+
+We first define the circuit of interest. For simplicity, in this example we use
+a randomized-benchmarking circuit whose ideal execution is equivalent to the
+identity operation.
+
+```{code-cell} ipython3
+from mitiq import benchmarks
+
+circuit = benchmarks.generate_rb_circuits(
+  n_qubits=1, num_cliffords=2, return_type = frontend,
+)[0]
+
+print(circuit)
+```
+
+## Fold the circuit
+In Mitiq, folding functions input a circuit and a scale factor, i.e., 
+a floating point value which corresponds to (approximately) how much the length 
+of the circuit is scaled. The minimum scale factor is one (which corresponds to folding no gates). 
+A scale factor of three corresponds to folding all gates locally. 
+Scale factors beyond three begin to fold gates more than once.
+
++++
+
+For local folding, there is a degree of freedom for which gates to fold first. 
+The order in which gates are folded can have an important effect on how the noise is scaled. 
+The function `fold_gates_from_left` will fold gates from the left (or start) of the 
+circuit until the desired scale factor is reached.
+
+```{code-cell} ipython3
+from mitiq.zne.scaling import fold_gates_from_left
+folded = fold_gates_from_left(circuit, scale_factor=2.)
+print("Folded circuit:", folded, sep="\n")
+```
+
+## Define the executor function
+We refer to a user-defined function that inputs a quantum program and executes it on a backend as an executor. 
+Such functions always accept a quantum program, sometimes accept other arguments, and always return an expectation value as a float. 
+As an example, below we define a simple executor function which inputs a Cirq circuit, executes it on a noisy simulator, and returns the probability of the ground state.
+
+```{code-cell} ipython3
+import numpy as np
+from cirq import Circuit, depolarize, DensityMatrixSimulator
+
+# initialize a backend
+SIMULATOR = DensityMatrixSimulator()
+# 5% depolarizing noise
+NOISE = 0.05
+
+
+def executor(circ: Circuit) -> float:
+   """Executes a circuit with depolarizing noise and
+   returns the expectation value of the projector |0><0|."""
+   circuit = circ.with_noise(depolarize(p=NOISE))
+   rho = SIMULATOR.simulate(circuit).final_density_matrix
+   obs = np.diag([1, 0])
+   expectation = np.real(np.trace(rho @ obs))
+   return expectation
+```
```

Comment:
This works only if `frontend = "cirq"`.  Not urgent, but it would be nice to make it universal. See e.g. pec-intro.myst in #1004 .  Feel free to copy and paste :-) 

Otherwise if we keep the current Cirq-only executor it is better to remove the `frontend` option in the first cell.

Reply:
Good catch, @andreamari. I intended to use the `frontend` option and then forgot to modify the rest of the implementation accordingly 🤦🏼‍♀️